src/webview/utils/ErrorHandler.ts

```
@@ -0,0 +1,198 @@
+      message: error.message,
+      context,
+      stack: error.stack,
+    });
+  }
+
+  /**
+   * ユーザーにエラーメッセージを表示
+   */
+  private showUserError(message: string): void {
+    // ステータスマネージャーが利用可能な場合は使用
+    if (typeof window !== 'undefined' && (window as any).statusManager) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)